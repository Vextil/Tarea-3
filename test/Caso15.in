# stress Set
# Prueba de orden de tiempo
TiempoSet
# Prueba de uso de memoria
MemoriaSet
Fin
