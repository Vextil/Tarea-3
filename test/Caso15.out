1># stress Set
2># Prueba de orden de tiempo
3>Fin operaciones. 
4># Prueba de uso de memoria
5>Fin Destruir. Fin Remover. Fin Operaciones. 
6>Fin
